src/app/services/api-file.service.ts

```diff
@@ -200,6 +196,41 @@ export class ApiFileService {
     );
   }
 
+  getAllAnalyses(query: any, size: number) {
+    const url = 'http://wp-np3-e2:9200/faang_build_1_analysis/_search/' + '?size=' + size;
```

Comment:
Is it for test purposes only? This should be changed to "this.hostSetting.host" as in commented line below.

Reply:
please check commit 71211ff